.github/workflows/nix-update.yml

```diff
@@ -0,0 +1,50 @@
+name: Update flake.nix to use latest version of ASIMOV CLI
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 16 * * 4"
```

Comment:
Always include a comment ("At 16:00 on Thursday") with a Crontab.guru link (https://crontab.guru/#0_16_*_*_4) for any crontab spec. Also, this should run more often than once a week. Make it nightly

Reply:
@artob it is done - https://github.com/asimov-platform/asimov-packaging/pull/5/commits/d14074162dc2a73ea115fd0a9988c8ce04acf89d